controllers/cache/labels.go

```diff
@@ -0,0 +1,42 @@
+package cache
+
+const (
+	AppLabel = "harbor-cluster-operator"
+)
+
+// NewLabels returns new labels
+func (redis *RedisReconciler) NewLabels() map[string]string {
+	dynLabels := map[string]string{
+		AppLabel:                       redis.Name,
```

Comment:
If we need to add some labels on component CRs, we should extract some const variable about the labels name.